common/test/run-param.c

```diff
@@ -183,7 +183,7 @@ static void tok_tok(void)
 	{
 		unsigned int n;
 		const jsmntok_t *tok = NULL;
-		struct json *j = json_parse(cmd, "{ 'satoshi', '546' }");
+		struct json *j = json_parse(cmd, "{ 'satoshi': '546' }");
```

Comment:
Interesting issue, I was not expecting `jsmn` to catch this correctly, given the overall quality of the library we've seen so far.

Also kudos to @rustyrussell for hunting this down :+1: